src/snowflake/snowpark/dataframe_writer.py

```diff
@@ -15,6 +15,7 @@
     add_api_call,
     dfw_collect_api_telemetry,
 )
+from snowflake.snowpark._internal.open_telemetry_prototype import open_telemetry
```

Comment:
why call the file prototype? Sounds discouraging :)

Reply:
I'll change it after fix all issues XD